src/serializers/index.ts

```diff
@@ -97,12 +92,16 @@ export const createSerializers = (
     opts.omitHeaderNames ?? DEFAULT_OMIT_HEADER_NAMES,
   );
 
+  const trim = trimmer({
+    depth: opts.maxObjectDepth ?? DEFAULT_MAX_OBJECT_DEPTH,
+  });
```

Comment:
Just thinking, does this technically allow `max object depth + 1` because it doesn't include the top-level property like `err` or `headers` in the calculation? Should we `- 1` to align?

Reply:
True good call, I've updated it 🙏